app/controllers/api/events_controller.rb

```diff
@@ -83,7 +83,7 @@ def set_event
   def create_event(params)
     @event.transaction do
       @event.release_date = Time.now unless @event.release_date
-      @event.set_image_filenames(params[:thumb_url], params[:poster_url])
+      @event.set_image_filenames(params[:thumb_url], params[:poster_url], params[:timeline_url], params[:thumbnails_url])
```

Comment:
Should be 

`@event.get_image_filenames(params[:thumb_url], params[:poster_url], params[:timeline_url], params[:thumbnails_url])`

Reply:
Oh! "set" is correct here, but by mistake, my commit changed the name of the method in the Event model to "get".  Fixed and updated this branch!